### DISCUSSION TITLE
How to create top languages card please guide me ?

### DISCUSSION THREAD

**Discussion by parmeshwar1820** [anuraghazra/github-readme-stats#2068] [Q&A] [answered]
How to create top languages card please guide me, also if possible tell me how to create a good readme

**Comment by DURGESH716** [comment, marked as answer]
Hello @parmeshwar1820
you can create your customized  readme page by following the below steps:-
1) open and fill the details on this website [https://rahuldkjain.github.io/gh-profile-readme-generator/](ReadMe Generator)
2) Copy the code and paste it somewhere in your local directory for later use.
3) Now, create a repository with your username referred to as "special repository"
4) Paste the code you copied earlier in the readme section of this special repository and click save.

**Reply by rickstaa** [reply]
Great answer thanks for jumping in on this one 🚀 . To add to @DURGESH716 answer:

@parmeshwar1820 Welcome to GRS. As explained in the [README](https://github.com/anuraghazra/github-readme-stats#top-languages-card) a language card can be created with the following command:

```md
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)

See https://github.com/abhisheknaiidu/awesome-github-profile-readme for some example READMEs you can fork. You can also checkout [my](https://github.com/rickstaa/rickstaa) or [anuraghazra](https://github.com/anuraghazra)'s README for an example GRS profile.
